core/src/hlist.rs

```diff
@@ -441,6 +442,8 @@ macro_rules! gen_inherent_methods {
             /// acc
             /// ```
             ///
+            /// [`Poly`]: ../traits/struct.Poly.html
```

Comment:
Perhaps at some point we should update these docs to strip these handwritten links, as rustc has supported intra-doc links since 1.48.

Reply:
Ah interesting...good point; https://blog.rust-lang.org/2020/11/19/Rust-1.48.html#easier-linking-in-rustdoc

```rust
pub mod foo {
    /// Some docs for `Foo`
    ///
    /// You may want to use `Foo` with [`Bar`](crate::bar::Bar).
    pub struct Foo;
}

pub mod bar {
    /// Some docs for `Bar`
    ///
    /// You may want to use `Bar` with [`crate::foo::Foo`].
    pub struct Bar;
}
```

Reply:
I copypasted this from `map` docs, no brain involved. Maybe you should open an issue about it or write PR to fix it if you really care.

Reply:
Yeah, no, it's just an off-hand comment, nothing specific to this PR at all